### ISSUE TITLE
Improve the alt text support in the Twitter plugin

### ISSUE THREAD

**Issue by alexwlchan** [alexwlchan/alexwlchan.net#268]
* [ ] Use the alt text if provided by the original tweet
* [x] Fail the build if alt text isn't provided
* [ ] Support alt text on multi-image tweets

**Comment by alexwlchan** [comment]
I'm going to solve this problem by not embedding tweets with images without alt text, or I'll solve it if I really really want to.
